nb-engine/nb-engine-core/src/main/java/io/nosqlbench/engine/api/activityapi/core/LegacyActivityType.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022-2023 nosqlbench
+ * Copyright (c) nosqlbench
```

Comment:
let's leave this copyright year intact

Reply:
We've been removing. It is actually irrelevant. You can't reset the year. The year is the original year of the overall product. Copyright protections are time limited and don't reset.